dev/tests/integration/testsuite/Magento/Customer/Model/ResourceModel/Grid/CollectionTest.php

```diff
@@ -54,7 +54,7 @@ public function testGetItemByIdForUpdateOnSchedule()
         $item = $this->targetObject->getItemById($newCustomer->getId());
         $this->assertNotEmpty($item);
         $this->assertSame($newCustomer->getEmail(), $item->getEmail());
-        $this->assertSame('test street test city Armed Forces Middle East 01001', $item->getBillingFull());
+        $this->assertSame('test street' . PHP_EOL . ' test city Armed Forces Middle East 01001', $item->getBillingFull());
```

Comment:
I think this case (and possibly others here) should be fixed to the test, rather than matching the test to the result. I'm pretty sure this was broken by a previous Mage-OS PR, https://github.com/mage-os/mageos-magento2/pull/117 . It shouldn't output a newline for empty address lines. Probably need to add a trim or array filter somewhere in there.